index.bs

```diff
@@ -5483,6 +5497,267 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 :: None.
 
 
+# Automation # {#sctn-automation}
+
+For the purposes of user-agent automation and application testing, this document defines a number of [=extension
+commands=] for the [[WebDriver]] specification. 
+
+## <dfn>Virtual Authenticators</dfn> ## {#sctn-automation-virtual-authenticators}
+
+These [=extension commands=] create and interact with [=Virtual Authenticators=]: software implementations of the
+<a>Authenticator Model</a>. <a>Virtual Authenticators</a> are stored in a <dfn>Virtual Authenticator Database</dfn> and
+have the following properties:
+
+: |authenticatorId|
+:: An arbitrary non-null string that uniquely identifies the [=Virtual Authenticator=].
+: |protocol|
+:: The protocol the [=Virtual Authenticator=] uses to encode its messages. Either `ctap2` for the [[FIDO-CTAP]] format
```

Comment:
> Also, various actual authenticators support multiple CTAP protocol versions. Ought we allow for instantiating virftAuthnrs (virtual authenticators) that also do so?

We can certainly do this. Do you see any test case where it might be useful? I had doubts about adding the protocol version in the first place because it should be transparent to the client of the API...